examples/sphere/shallow_water.jl

```diff
@@ -354,7 +354,7 @@ else # steady-state and mountain test cases share the same form of fields
     end
 end
 
-function rhs!(dydt, y, parameters, t)
+function rhs!(dYdt, y, parameters, t)
```

Comment:
Wouldn't we also want to change the second argument, `y` -> `Y` to be consisten?

Reply:
Ah, I thought I did. We can change this too. It's technically okay because of the scope, it's the global variable names that need to be consistent for the perf monitoring.